tests/config/test_config_comparator.py

```diff
@@ -54,41 +57,119 @@ def test_config_comparator():
+    assert blocked_config_diff["modified_items"][7] == (
         ("section_name", "section_5", "attribute"),
         (["1:int", "2:int", "3:int", "4:int", "5:int"], ["1:int", "2:int"]),
     )
 
-    assert len(config_diff["removed_items"]) == 1
-    assert config_diff["removed_items"][0] == (
+    assert len(blocked_config_diff["removed_items"]) == 1
+    assert blocked_config_diff["removed_items"][0] == (
         ("section_name", "section_3", None),
         {"attribute": "3:int", "prop": "prop_3"},
     )
+
+
+def test_unblock_section_when_compare(caplog):
+    unique_section_1 = UniqueSectionForTest(attribute="unique_attribute_1", prop="unique_prop_1")
+    section_1 = SectionForTest("section_1", attribute="attribute_1", prop="prop_1")
+    section_2 = SectionForTest("section_2", attribute=2, prop="prop_2")
+
+    _config_1 = _Config._default_config()
+    _config_1._sections[SectionForTest.name] = {"section_1": section_1}
+    _config_1._unique_sections[UniqueSectionForTest.name] = unique_section_1
+
+    _config_2 = _Config._default_config()
+    _config_2._global_config = GlobalAppConfig.default_config()
+    _config_2._global_config.repository_properties = {"foo": "bar"}
+    _config_2._sections[SectionForTest.name] = {"section_1": section_2}
+    _config_2._unique_sections[UniqueSectionForTest.name] = unique_section_1
+
+    # Compare 2 Configuration
+    config_diff = _ConfigComparator(_config_1, _config_2)
+
+    config_diff._log_unblocked_sections()
+    assert caplog.text == ""
+    assert config_diff.get("unblocked_diff_sections") is None
+    caplog.clear()
+
+    blocked_config_diff = config_diff["blocked_diff_sections"]
+    config_diff._log_blocked_sections()
+    assert len(caplog.text.strip().split("\n")) == 3
+    assert len(blocked_config_diff["added_items"]) == 1
+    assert len(blocked_config_diff["modified_items"]) == 2
+    assert blocked_config_diff.get("removed_items") is None
+    caplog.clear()
+
+    # Ignore any diff of "section_name" and compare
+    _ConfigComparator._unblock_section("section_name")
+    config_diff = _ConfigComparator(_config_1, _config_2)
+
+    config_diff._log_blocked_sections()
+    blocked_config_diff = config_diff["blocked_diff_sections"]
+    assert len(caplog.text.strip().split("\n")) == 1
+    assert len(blocked_config_diff["added_items"]) == 1
+    assert blocked_config_diff.get("modified_items") is None
+    assert blocked_config_diff.get("removed_items") is None
+    caplog.clear()
+
+    config_diff._log_unblocked_sections()
+    unblocked_config_diff = config_diff["unblocked_diff_sections"]
+    assert len(caplog.text.strip().split("\n")) == 2
+    assert unblocked_config_diff.get("added_items") is None
+    assert len(unblocked_config_diff["modified_items"]) == 2
+    assert unblocked_config_diff.get("removed_items") is None
+    caplog.clear()
+
+    # Ignore any diff of Global Config and compare
+    _ConfigComparator._unblock_section(["TAIPY"])
```

Comment:
Is there a use case where GlobalAppConfig is blocking? 
If not, this should be the default behavior, and it should be tested.

Reply:
IMO, any change on the GlobalAppConfig should block the application.

Between versions, updates on `root_folder`, `repository_type`, `repository_properties`, `clean_entities_enabled` are all blocking changes.

A special case is `storage_folder`, on FS repository, change `storage_folder` and the application can not detect older versions anymore.